Sources/ConsoleKit/Command/CommandError.swift

```diff
@@ -1,47 +1,44 @@
 /// Errors working with the `Command` module.
-public struct CommandError: Error, CustomStringConvertible {
-    /// See `Debuggable`.
-    public let identifier: String
-
-    /// See `Debuggable`.
-    public let reason: String
-    
-    /// The name of the attempted command
-    public let command: String?
-    
-    /// All available commands
-    public let availableCommands: [String]
+public enum CommandError: Error, CustomStringConvertible {
```

Comment:
I think it would make sense to conform this enum to `Equatable`. It might make your error tests more concise, and with this being a public type, we don't know what dependents on this package will want to do. I expect the compiler will be able to synthesize the conformance, so it shouldn't be too hard.
```suggestion
public enum CommandError: Error, Equatable, CustomStringConvertible {
```